### DISCUSSION TITLE
How good does the miniDexed sound?

### DISCUSSION THREAD

**Discussion by Banana71** [probonopd/MiniDexed#445] [Show and tell]
A sound comparison between miniDexed and Yamaha SY99.
The document is currently available in German.

[miniDexed Comparsion.pdf](https://github.com/probonopd/MiniDexed/files/10889318/miniDexed.Comparsion.V1.1.pdf)

![Cover small](https://user-images.githubusercontent.com/104296205/222085308-ae7ce000-dfbb-4aad-86f5-7f4315c0b4dc.jpg)

**Comment by Banana71** [comment]
Full resolution

sounds and looks the same
![SPAN C1 C4](https://user-images.githubusercontent.com/104296205/222085534-b48e15fb-921a-4f05-891a-9516a60e91cc.jpg)


the miniDexed has a 8 ms higher latency than the Yamaha SY99
![Latenz MIDI](https://user-images.githubusercontent.com/104296205/222085570-61396499-9f35-4859-8c08-a2dc9c58c90c.jpg)

**Reply by DutchBigEd** [reply]
Thank you, great readig stuff!
